package/CHANGELOG

```diff
@@ -18,6 +18,8 @@ The rules for this file:
  * 2.5.0
 
 Fixes
+  * np.histogramdd calls in :class:`DensityAnalysis` now pass the `density`
```

Comment:
@MDAnalysis/coredevs please look at discord for this one - in my opinion we probably should do a 2.4.2 release since this has effectively killed the DensityAnalysis code. However, _how_ we do bugfix releases really should be discussed.

TLDR; I'm pushing for us to be able to do bugfixes from a separate feature branch than `develop`. That way PRs can keep rolling into `develop` without breaking semver if we need to do a bugfix release.